### DISCUSSION TITLE
METER

### DISCUSSION THREAD

**Discussion by olegkaras2** [Bodmer/TFT_eSPI#1564] [Q&A - General]
https://drive.google.com/file/d/17Stx2QjPTsgRap-ur5HfeL5wv3vLzPD0/view?usp=sharing
Good afternoon.
I want to draw 2 counters, but nothing works, do you have a mayor where there are two counters? I can’t set the coordinates correctly. I want to make an ammeter + voltmeter

**Comment by Bodmer** [comment]
The simplest solution is to use the viewport features  (see examples) as you can move the coordinate datum.

You will need to add some variables, or duplicate the code in a new function for the second meter.

**Comment by olegkaras2** [comment]
Could you please specify which example. sorry, it's hard for me to understand, I use a translator

**Reply by Bodmer** [reply]
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/Generic/Viewport_Demo/Viewport_Demo.ino

**Comment by Bodmer** [comment]
I have adapted one of the examples to help you:
[Dual_TFT_Meter.zip](https://github.com/Bodmer/TFT_eSPI/files/7885254/Dual_TFT_Meter.zip)

This uses the viewport to move the coordinate datum.   To reset the viewport to the normal TFT full screen use:
  tft.resetViewport();

**Comment by olegkaras2** [comment]
Thank you very much, everything works.

**Comment by olegkaras2** [comment]
Good afternoon.
Please help to draw two independent colored needles for one device, I want
to display the set volts and amperes and measured volts and amperes.

вт, 18 янв. 2022 г. в 21:10, Олег Карась ***@***.***>:
